odoo/orm/models.py

```diff
@@ -3240,7 +3240,7 @@ def _init_column(self, column_name):
             value = None
         # Write value if non-NULL, except for booleans for which False means
         # the same as NULL - this saves us an expensive query on large tables.
```

Comment:
maybe also refactor this comment a little to make it consistent with the code. But since it is for master, why not directly raise an error for `required` boolean field?